scripts/h2/upgrade-0.6.0-to-0.7.0-h2.sql

```diff
@@ -25,11 +25,11 @@ CREATE TABLE IF NOT EXISTS `table_column_version_info` (
     `table_version` INT UNSIGNED NOT NULL COMMENT 'table version',
     `column_id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'column id',
     `column_name` VARCHAR(128) NOT NULL COMMENT 'column name',
-    `column_type` VARCHAR(128) NOT NULL COMMENT 'column type',
+    `column_type` TEXT NOT NULL COMMENT 'column type',
```

Comment:
When executing the upgrade SQL, this change will not work because the table `table_column_version_info` already exists

Reply:
No, this doesn't exist, this table is newly added in 0.7.0.